Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -21,9 +47,28 @@ def model(parameters):
     sys.stdout = sys.__stdout__ # Re-enable printing
     print("Simulation completed")
     
-    # Obtain summary statistics from simulation
-    dummy_result = np.array([parameters["gamma"], parameters["reg_param"]]) #Just return parameter values until summary statistics are working
-    return {"data": dummy_result}
+    # Load cellStates
+    pickle_list = helperFunctions.create_pickle_list(export_path)
+    cells = helperFunctions.load_cellStates(export_path, pickle_list[-1]) # load last pickle file
+    
+    # Extract time parameters from simulation (could hard-code to avoid unnecessary repetition)
+    sim_file = os.path.abspath(cellmodeller_module)
+    dt = sim_time_parameters(sim_file)
```

Comment:
Why do you need to search "dt" through the simulation file? this way you add extra lines to the code plus the code spends time fetching "dt". instead, you can define "dt" in the main function as a global variable and let a user define it.

Reply:
I think it will be preferable to leave the specification of dt to the simulation code (and developers thereof). That's a consideration that's separate from what this code is looking to accomplish.

Reply:
sim_time_parameters **fetches** the time step specified in the cellmodeller simulation module (e.g. see dt in simulation_module.py); it does not set the time step used by the simulation. I wrote this to prevent us forgetting to change dt in the pyabc file if it changes in the simulation module. Brian, I believe this function is in line with your comment, so I will leave the code as is.